documentation/guides/migrating-from-v4-to-v5.md

```diff
@@ -0,0 +1,156 @@
++ import '@shopify/polaris/dist/styles.css';
+```
+
+Our minified styles file has been removed, and you should use the above file instead.
+
+```diff
+- import '@shopify/polaris/styles.min.css';
++ import '@shopify/polaris/dist/styles.css';
+```
+
+Our scss entrypoint has been removed, and you should use the above styles file instead. If you require our mixins/functions you should also import the our public-api entrypoint.
+
+```diff
+- @import 'path_to_node_modules/@shopify/polaris/styles';
++ @import 'path_to_node_modules/@shopify/polaris/dist/styles.css';
++ /* Optional - only needed if you use our mixins/functions */
++ @import 'path_to_node_modules/@shopify/polaris/dist/styles/public-api';
+```
+
+## Build output changes - @shopify/sewing-kit integrations only
+
+Apps using `@shopify/sewing-kit` - Shopify's in-house opinionated build toolkit - will need to update to at least v0.132.2 to make use of the reworked "esnext" build output in polaris v5.
+
+Apps using `@shopify/sewing-kit` and Polaris v4 also had to have an explicit import of `@shopify/polaris/esnext/styles/global.scss` in the same file as they imported and used `AppProvider`. This is no longer required, this import can be removed. The styles in this file will be implicitly imported when you use `AppProvider`.
+
+```diff
+- import '@shopify/polaris/esnext/styles/global.scss';
+```
+
+Sewing-kit apps can use our sass public API by configuring their sass plugin's autoInclude option. The path to the sass API file has now changed and should be updated accordingly:
+
+```diff
+plugins.sass({
+  autoInclude: [
+-    path.join(__dirname, 'node_modules/@shopify/polaris/esnext/styles/_public-api.scss'),
++    path.join(__dirname, 'node_modules/@shopify/polaris/dist/styles/_public-api.scss'),
+  ],
+ })
+```
+
+## Component API Changes
+
+### ResourceList.FilterControl
+
+The `ResourceList.FilterControl` subcomponent has been removed. You should use the `Filters` component to provide filtering options.
+
+### EmptyState
+
+The `EmptyState` component's `centeredLayout` prop has been removed as the illustration's location is now handled automatically by opting into the new design language. You should remove any usage of this prop.
+
+### Removed Exports
+
+#### NavigationMessageProps
+
+The `NavigationMessageProps` type has been removed, as Navigation's Message internal subcomponent has been removed.
+
+## Dependencies
+
+The peer dependencies on `react` and `react-dom` have been increased to 16.9.0, to allow us to use react's Profiler component in the future. Use the package manager of your choice to install a recent version of these pacakges.
+
+Apps built using `@shopify/sewing-kit` - Shopify's in-house opinionated build toolkit - will need to upgrade to at least version `0.132.2`.
+
+## CDN usage
+
+As a result of build output changes, the location of our static stylesheet for CSS-only usage has been moved. Update any stylesheet imports to use the new path:
```

Comment:
I try to avoid "our", in case it's perceived as "not yours".

```suggestion
As a result of build output changes, the location of the static stylesheet for CSS-only usage has been moved. Update any stylesheet imports to use the new path:
```